html/wifi-networks.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>WIfi Networks</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+    </head>
+
+    <body>
+        {{template "navbar"}}
+
+
+
+        <div class="container">
+
+            {{if eq .Error ""}}
+            <div id="errorAlert" class="alert alert-warning alert-dismissible" style="visibility: hidden">
+            {{else}}
+            <div id="errorAlert" class="alert alert-warning alert-dismissible" style="">
+            {{end}}
+              <a href="#" onclick="closeAlert()"  class="close"  aria-label="close">&times;</a>
+              <strong><span id="errorSpan">{{.Error}}</span></strong> 
+            </div>
```

Comment:
David's PR landed yesterday. Can you pls make sure that the error alert there uses a similar approach to what you've done here. Separate PR to make them consistent is fine.